### ISSUE TITLE
FontAwesome CSS

### ISSUE THREAD

**Issue by leeroybrun** [kmohrf/connect-fonts-fontawesome#3]
How do you handle the FontAwesome CSS ? This plugin will just handle the fonts, but you have to add the CSS somewhere else in your project. 
So when FontAwesome is updated you have to manually update the CSS, right ?

**Comment by kmohrf** [comment]
yes. my setup has a custom build setup that automatically builds a less file. the package could provide a less file including all less modules from fontawesome without the one that contains the font-face declaration. i guess that would help with some setups.

any other suggestions?

**Comment by leeroybrun** [comment]
I don't know if we can customize the CSS returned by connect-fonts. But maybe the ideal would be to generate the CSS from the fontawesome LESS files (without the one containing the font-face declaration) and append this to the CSS generated by connect-fonts.

What do you think ? This way we will have the complete CSS ready to use.

**Comment by saper** [comment]
Yes, I bumped into this problem too.
